CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ include(CheckCXXCompiler.cmake)
 include(CheckIncludeFileCXX)
 include(FindPythonModule)
 
+# Precompiled headers support. Till cmake 3.16 becomes available use cotire.
+include(cotire)
```

Comment:
To generate precompiled header. Might speed up build during development. Precompiled header supports is added in cmake version 3.16. When this version is widely available, drop this module.